third_party/miniz-2.0.8/miniz.c

```diff
@@ -4522,7 +4522,9 @@ void *mz_zip_reader_extract_file_to_heap(mz_zip_archive *pZip, const char *pFile
 mz_bool mz_zip_reader_extract_to_callback(mz_zip_archive *pZip, mz_uint file_index, mz_file_write_func pCallback, void *pOpaque, mz_uint flags)
 {
     int status = TINFL_STATUS_DONE;
+#ifndef MINIZ_DISABLE_ZIP_READER_CRC32_CHECKS
```

Comment:
This is a fix that was upstreamed in [this PR](https://github.com/richgel999/miniz/pull/143)

Reply:
Are we going to update miniz at some point?

Reply:
We could but I think the update would just be the same as this change (plus whatever has been added since)